src/Shared/interfaces.ts

```diff
@@ -191,6 +191,8 @@ export interface PageMeta {
   id: number;
   createdAt: string;
   updatedAt: string;
+  hasEditReview: false;
```

Comment:
This is the interface for the Pages edit screen metabox. So we do not need to make any changes here.